### ISSUE TITLE
Multiple function indentation issue

### ISSUE THREAD

**Issue by eddiezane** [pangloss/vim-javascript#486]
This is on the `develop` branch.

It appears `jobs.on` isn't indenting correctly in this snippet.

I'll try to whip up and drill down a minimal case later but wanted to report now.

![screen shot 2016-06-16 at 5 08 44 pm](https://cloud.githubusercontent.com/assets/959573/16133309/1b3d1d98-33e5-11e6-92b9-691191fbd182.png)

```javascript
app.get('/convert', (req, res) => {
  const { url, options } = req.query
  console.log(options)
  const job = queue.createJob('photo', {
    url,
    options
  })
  .save(err => {
    if (err) {
      console.error(err)
    }
  })

job.on('complete', result => {
  fs.createReadStream(result)
    .pipe(res)
})
})
```

**Comment by eddiezane** [comment]
Seems to happen only with the chained `.save` present.

**Comment by eddiezane** [comment]
Closed with #488

Nice fix, @bounceme!
